master/static/migrations/20240409101349_add-arbitrary-metadata-table.tx.up.sql

```diff
@@ -0,0 +1,32 @@
+-- ALTER RUNS TABLE
+ALTER TABLE runs ADD COLUMN metadata JSONB;
```

Comment:
If anyone logs anything slightly large too this field (well, especially medium-sized when it misses TOAST), I think the performance is going to be horrendous. And I think Python API will lead users into behavior.